timemachine/fe/rbfe.py

```diff
@@ -630,7 +630,7 @@ def run_solvent(
         steps_per_frame=steps_per_frame,
         min_cutoff=min_cutoff,
     )
-    return solvent_res, solvent_top
+    return solvent_res, solvent_top, solvent_host_config
```

Comment:
I think https://github.com/proteneer/timemachine/blob/73ea83a18de73b5bd6a2836ef6828d4eb20082d8/tests/test_relative_free_energy.py#L193 needs to be updated to match